xapian-core/include/xapian/cluster.h

```diff
@@ -162,15 +163,20 @@ class XAPIAN_VISIBILITY_DEFAULT TermListGroup : public FreqSource {
      *
      *  @param document 	Adds a document and updates the TermListGroup
      *				based on the terms found in the document
+     *  @param stopper		Xapian::Stopper object to identify stopwords
      */
-    void add_document(const Document &document);
+    void add_document(const Document &document, const Stopper *stopper = NULL);
 
   public:
     /** Constructor
      *
      *  @param docs	MSet object used to construct the TermListGroup
+     *  @param stopper	Xapian::Stopper object to identify stopwords
      */
-    explicit TermListGroup(const MSet &docs);
+    explicit TermListGroup(const MSet &docs, const Stopper *stopper = NULL);
+
+    /// Default constructor
+    TermListGroup();
```

Comment:
Why did you add a default constructor?  It doesn't seem useful for this class (if you really wanted an empty `TermListGroup` for some reason, `TermListGroup(MSet())` would give you one).